backend/Dockerfile

```diff
@@ -1,5 +1,5 @@
 FROM node:8.10
-RUN mkdir /backend
-ADD . /backend
-WORKDIR /backend
-RUN npm install
\ No newline at end of file
+WORKDIR /app
+ADD . /app
+RUN npm install
+RUN npm install -g nodemon
```

Comment:
I think `nodemon` should be a dev dependency and then used via npm scripts instead of using global scripts.

Reply:
it's contained inside the container so I don't see why we have to bother going around the hard way around and do
start: ./node_modules/.bin/nodemon src/start.js